scripts/MRK17_Exp1_rk00.jmd

```diff
@@ -18,7 +18,7 @@ using CSV, DataFrames, MixedModels
 
 ```julia
 dat = CSV.read(datadir("MRK17_Exp1_xtra.csv"));
-describe(dat)
+describe(dat)   # why are there numeric encodings of factors in this dataset?
```

Comment:
I really want to avoid the practice of using numeric codings for factors.  It is dangerous.